tests/openassetio/test/managerValidator/test_commandLine.py

```diff
@@ -0,0 +1,128 @@
+import json
```

Comment:
[fix] #141 will remove all `pylint` warnings for the test code base. Can we apply anything relevant from that PR here.

Reply:
Did some pylinting in d9addd2